src/library.js

```diff
@@ -4423,6 +4423,9 @@ LibraryManager.library = {
   // The exact return value depends in whether we are running WASM or JS, and whether
   // the engine supports offsets into WASM. See the function body for details.
   emscripten_generate_pc: function(frame) {
+#if !USE_OFFSET_CONVERTER
+    abort('Cannot use emscripten_generate_pc (needed by __builtin_return_address) without -s USE_OFFSET_CONVERTER');
+#endif
```

Comment:
Should this abort be moved down into the condition for old v8 so users on other platforms aren't penalized and so we can remove this once old v8 is old enough?

Reply:
This is a correctness issue. If you test on a new v8 (or firefox), you might miss `-s USE_OFFSET_CONVERTER` when using `__builtin_return_address`, and then not realize the problem until someone else comes along with an old v8.

Reply:
Yeah I guess that's not great. Somewhat unfortunate in terms of shipping down (sometimes) unnecessary code, though.